apps/sms/views/conversation/js/conversation.js

```diff
@@ -3015,6 +3022,37 @@ var ConversationView = {
     threadMessagesClass.remove('editable-select-mode');
     node.focus();
     window.getSelection().selectAllChildren(node);
+  },
+
+  initLowStorageHandling: function() {
```

Comment:
__thinking aloud:__ I don't know why exactly yet :), but I feel that ```App.js``` can work with ```DeviceStorageClient``` instead (or even replace it, not sure on this one), as it seems app specific thing. Then app.js can append specific CSS class to the body as it does with ```.js-app-ready``` (in this case we can remove similar code from Inbox entirely) and broadcast event if anyone else needs more work (like ConversationView here).

Just thought.